Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -844,7 +844,8 @@ JS::NonnullGCPtr<Geometry::DOMRect> Element::get_bounding_client_rect() const
         return Geometry::DOMRect::create(realm(), absolute_rect.to_type<float>());
     }
 
-    if (auto const* paintable = this->paintable(); is<Painting::InlinePaintable>(*paintable)) {
+    auto const* paintable = this->paintable();
+    if (paintable && is<Painting::InlinePaintable>(*paintable)) {
```

Comment:
Since we don't actually need `paintable` to live longer than before, this could be written:

```cpp
    if (auto const* paintable = this->paintable(); paintable && is<Painting::InlinePaintable>(*paintable)) {
```

Reply:
it is also used to decide whether FIXME message needs to be printed or not.

Reply:
Ah in the next commit. Fair!